html-web-page/index.html

```diff
@@ -0,0 +1,108 @@
+
+    td:not(:first-child) {
+      text-align: center;
+    }
+  </style>
+</head>
+<body>
+
+  <!-- your HTML code here 👇 -->
+  <div class="jotmc-title">
+    <h1>Jelly of the Month Club</h1>
+  </div>
+  <div class="jotmc-slogan">
+    <h2>Make all your friends jelly of your stock of jelly!</h2>
+  </div>
+  <div class="jotmc-description">
+    <p>Here at JOTMC Inc., we are constantly striving to supply our customers
+      with the highest&dash;quality and most unique jellies on a monthly basis.
+      For the jelly lover in your life &lpar;or yourself!&rpar;, this is the
+      gift that keeps on giving all year round. We offer three tiers of monthly
+      jelly joy, and each one offers their own perks. Here is a breakdown of
+      each, as well as your monthly price&colon;
+    </p>
+  </div>
+  <div class="jotmc-info">
+    <table class="jotmc-info-table">
+      <thead>
+        <tr>
+          <th></th>
+          <th>Basic Plan</th>
+          <th>Delux Plan</th>
+          <th>Jelly Giant Plan</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>Three new curated jellies per month</td>
+          <td><strong>&check;</strong></td>
+          <td><strong>&check;</strong></td>
+          <td><strong>&check;</strong></td>
+        </tr>
+        <tr>
+          <td>Monthly jellies come with commemorative mason jars</td>
+          <td></td>
+          <td><strong>&check;</strong></td>
+          <td><strong>&check;</strong></td>
+        </tr>
+        <tr>
+          <td>Free bonus jar of preserves bi-monthly</td>
+          <td></td>
+          <td><strong>&check;</strong></td>
+          <td><strong>&check;</strong></td>
+        </tr>
+        <tr>
+          <td>Free yearly tickets to Jelly Con</td>
+          <td></td>
+          <td></td>
+          <td><strong>&check;</strong></td>
+        </tr>
+      </tbody>
+      <tfoot>
+        <tr>
+          <td>Monthly Pricing</td>
+          <td>&dollar;30</td>
+          <td>&dollar;50</td>
+          <td>&dollar;60</td>
+        </tr>
+      </tfoot>
+    </table>
+  </div>
+  <div class="jotmc-reg">
+    <p>Like what you see? Become a part of the JOTMC family today!</p>
+    <form
+      action="file:///C:/Users/Owen/repos/m1-0122-code-solutions/html-web-page/index.html"
+      method="get">
+      <div class="jotmc-user-input">
+        <label for="user-name">
+          <input type="text" name="user-name" id="user-name" placeholder="Your Name" required="required">
+        </label>
+        <label for="plan-selection">
+          <select name="plan" id="plan-selection" required="required">
+            <option value="Select Your Plan&colon;" disabled>Select Your Plan&colon;</option>
+            <option value="Basic Plan">Basic Plan</option>
+            <option value="Deluxe Plan">Delux Plan</option>
+            <option value="Jelly Giant Plan">Jelly Giant Plan</option>
+          </select>
+        </label>
+        <label for="submit-user-info">
```

Comment:
the only thing to remove is the label for the submit input, you don't want them on submit inputs: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#buttons